.github/workflows/tag_image_push.yml

```diff
@@ -28,37 +32,44 @@ jobs:
         with:
           version: v0.5.1
 
-
       - name: Login to Docker Hub
         run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
 
       - name: Build/Push OSS image
         run: |
+          TAGS="--tag hazelcast/hazelcast:${{ env.RELEASE_VERSION }}"
+          if [[ "${{ env.PUSH_LATEST }}" == "yes" ]]; then
+            TAGS="${TAGS} --tag hazelcast/hazelcast:latest"
+          fi
           docker buildx build --push \
-            --tag hazelcast/hazelcast:${{ env.RELEASE_VERSION }} \
+            --build-arg HZ_VERSION=${{ env.RELEASE_VERSION }} \
+            ${TAGS} \
             --platform=linux/arm64,linux/amd64,linux/ppc64le,linux/s390x hazelcast-oss
+
       - name: Build/Push EE image
         run: |
+          TAGS="--tag hazelcast/hazelcast:${{ env.RELEASE_VERSION }}"
```

Comment:
I guess it should be `hazelcast-enterprise`

Reply:
Yes you are right, when I was testing I commented out these parts. I will update them.